_assets/docs/DEPENDENCIES.md

```diff
@@ -49,10 +49,10 @@ file is changed, `dep ensure` will re-generate `Gopkg.lock` as well.
 
 ## Updating `Geth`
 
-1. Update `develop` branch in [`status-im/go-ethereum`](https://github.com/status-im/go-ethereum/tree/develop).
-2. Update the `go-ethereum` dependency: `dep ensure -v -update github.com/ethereum/go-ethereum`.
-3. Make sure that `[[constraint]]` statements in `status-go/Gopkg.toml` contains the same SHAs as `go-ethereum/vendor/vendor.json`.
-4. Update vendor files in `status-go`, running `dep ensure`.
+1. Update the `go-ethereum` dependency: `dep ensure -v -update github.com/ethereum/go-ethereum`.
+2. Apply patches using [the patcher script](../patches/patcher). Update patches to resolve conflicts if needed.
+3. Update vendor files in `status-go`, running `dep ensure`.
```

Comment:
Do we want to reference instead `make dep-ensure`?

Reply:
agree!